api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const baseURL = "https://nc-news-o4n5.onrender.com/api/"
+
+export const fetchArticles = () => {
+return axios.get(`${baseURL}articles`).then(({data}) => {
+    return data.articles
+})
+}
```

Comment:
You don't need to explicitly add the baseURL in the request as axios does that for you. Your request can look like `axios.get("/articles")` 🙂